mantidproject/mantid
# Add h5py to osx for homebrew & macports

We weren't shipping `h5py` on OSX. This PR resolves this. 

Fixes #16909 

**To test:**
- Make sure that an app bundle for OSX is created.
1. Install `h5py` on your local OSX machine (following the OSX build instruction)
2. Create a package on your local machine
3. Install the Mantid package and import `h5py` in the python console



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
